front/src/components/Dialogs/TaskExportDialog.vue

```diff
@@ -104,32 +137,60 @@ export default defineComponent({
         )
       ).join('\n\n');
 
-      // download
       const blob = new Blob([template], {
         type: 'text/markdown',
       });
-      const link = document.createElement('a');
-      link.href = URL.createObjectURL(blob);
-      link.download = this.ctf.title + '.md';
-      link.click();
-      URL.revokeObjectURL(link.href);
+
+      this.downloadBlob(blob);
+    },
+
+    async exportTasksAsZip(tasks: Task[]) {
+      const zip = new JSZip.default();
+
+      const files = await Promise.all(
+        tasks.flatMap(async (task) => {
+          return {
+            title: task.title,
+            content: await this.downloadTaskMarkdown(task),
+          };
+        }, this)
+      );
+
+      zip.file(
+        `${this.ctf.title} - Overview.md`,
```

Comment:
Also here, the ctf title might contain illegal character for filename. I don't know how jslib handles it and if we really need to care about this.

Reply:
See this comment: https://github.com/TFNS/CTFNote/pull/204#discussion_r1107631863

Reply:
Sounds good